data/json/items/comestibles/egg.json

```
@@ -1257,5 +1257,55 @@
     "name": { "str": "cluster of frog's eggs", "str_pl": "clusters of frog's eggs" },
     "rot_spawn": "GROUP_EGG_SHIFT_FROG",
     "vitamins": [ [ "iron", 4 ], [ "mutant_toxin", 8 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_proxy_bigfrog",
+    "copy-from": "egg_proxy_frog",
+    "name": { "str": "cluster of giant frog's eggs", "str_pl": "clusters of giant frog's eggs" },
+    "weight": "160 g",
+    "volume": "1 L",
+    "symbol": "O",
+    "quench": 8,
+    "calories": 200,
+    "description": "A large clump of very large frog eggs.  They're gelatinous, transparent and larger even than chicken's eggs. ",
```

```suggestion
    "description": "A large clump of very large frog eggs.  They're gelatinous, transparent and larger than even chicken's eggs. ",
```
(or remove the word even entirely)